etl/resources/arkg_config.py

```diff
@@ -0,0 +1,9 @@
+from dagster import ConfigurableResource
+
+from etl.models.types import RdfMimeType
+
+
+class ArkgConfig(ConfigurableResource):  # type: ignore[misc]
+    """A ConfigurableResource that contains values used by an ARKG."""
+
+    rdf_mime_type: RdfMimeType
```

Comment:
This is another situation where you want to build out assets rather than have configuration change the pipeline output.

Reply:
I have added `wikipedia_arkg_factory` to build different assets based on an RDF serialization.